src/domain/event/TicketmasterEventIsEnrolled.tsx

```diff
@@ -0,0 +1,75 @@
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import { useQuery } from '@apollo/client';
+
+import styles from './ticketmasterEventIsEnrolled.module.scss';
+import { ticketmasterEventQuery } from './queries/ticketmasterEventQuery';
+import {
+  ticketmasterEventQuery as TicketmasterEventQueryType,
+  // eslint-disable-next-line max-len
+  ticketmasterEventQuery_event_ticketSystem_TicketmasterEventTicketSystem as EventTicketSystem,
+  // eslint-disable-next-line max-len
+  ticketmasterEventQuery_event_occurrences_edges_node_ticketSystem_TicketmasterOccurrenceTicketSystem as OccurrenceTicketSystem,
+} from '../api/generatedTypes/ticketmasterEventQuery';
+import LoadingSpinner from '../../common/components/spinner/LoadingSpinner';
+import Paragraph from '../../common/components/paragraph/Paragraph';
+import EventPage from './EventPage';
+import SuccessToast from './enrol/SuccessToast';
+import ErrorMessage from '../../common/components/error/Error';
+import { useChildRouteGoBackTo } from '../profile/route/ProfileRoute';
+import AnchorButton from '../../common/components/button/AnchorButton';
+import TicketmasterPassword from './TicketmasterPassword';
+import Text from '../../common/components/text/Text';
+
+const TicketmasterEventIsEnrolled = () => {
+  const { t } = useTranslation();
+  const goBackTo = useChildRouteGoBackTo();
+  const params = useParams<{ childId: string; eventId: string }>();
+  const { loading, error, data } = useQuery<TicketmasterEventQueryType>(
+    ticketmasterEventQuery,
+    {
+      variables: {
+        eventId: params.eventId,
+        childId: params.childId,
+      },
+    }
+  );
+
+  if (loading) return <LoadingSpinner isLoading={true} />;
+
+  const errorMessage = <ErrorMessage message={t('api.errorMessage')} />;
+  if (error || !data?.event) return errorMessage;
+
+  const eventTicketSystem = data?.event.ticketSystem as EventTicketSystem;
+  /* TODO Currently we don't have Ticketmaster URLs for whole events, so as a workaround
+      the URL of the first occurrence is used for now.
+   */
+  const ticketmasterUrl = (
+    data?.event?.occurrences?.edges[0]?.node
+      ?.ticketSystem as OccurrenceTicketSystem
+  ).url;
+
+  return (
+    <EventPage event={data.event} success={<SuccessToast />} backTo={goBackTo}>
+      <div className={styles.description}>
+        <Paragraph text={data.event.description || ''} />
+      </div>
+      <Text>{t('ticketmasterEvent.description')}</Text>
+      <Text>{t('ticketmasterEvent.passwordLabel')}</Text>
+      <div className={styles.passwordRow}>
+        <TicketmasterPassword password={eventTicketSystem?.childPassword} />
+        <AnchorButton href={ticketmasterUrl}>
```

Comment:
This should take the user to a new tab. Also it should tell about that for a screen reader.

Reply:
I'm not sure, but probably this can take `target="_black"` and `aria-label={ t('common.openInNewTabAriaLabel')}` as an attribute.

Reply:
I think there should also be an "external link" -icon in it.

Reply:
Not sure about this, in [HDS principles](https://hds.hel.fi/components/link#principles) it says _Generally, links should not open in a new tab_.

Reply:
It also says that 

> HDS Link supports a link with an external icon. Use the external prop when you wish to navigate to an entirely different site.

and (https://hds.hel.fi/components/link/accessibility) 

> Remember to tell users if the link is going to open an entirely new site. In HDS React Link this feature is provided through external links (prop external). There is a default aria-label for screen readers. This can be overwritten by using the openInExternalDomainAriaLabel prop. In HDS Core, set a proper aria-label to the anchor tag as shown in the examples below.

and 

> Remember to tell users if the link is going to open in a new tab. In HDS React Link this feature is provided through prop openInNewTab. There is a default aria-label for screen readers. This can be overwritten by using the openInNewTabAriaLabel prop. In HDS Core, set a proper aria-label to the anchor tag as shown in the examples below.

Also we get some rejecting comments from auditors when an accessibility audit was made to Tapahtumat.

Reply:
Added the external link icon and changed the thing to open in a new tab.